src/client/Microsoft.Identity.Client/Internal/Logger/ILoggerAdapter.cs

```diff
@@ -1,41 +1,25 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
-using System.Diagnostics;
 using System.Runtime.CompilerServices;
 using Microsoft.Identity.Client.Internal.Logger;
+using Microsoft.IdentityModel.Abstractions;
 
 namespace Microsoft.Identity.Client.Core
 {
-    internal interface ICoreLogger
+    internal interface ILoggerAdapter
```

Comment:
Well, the `LoggerAdapters` aren't actually loggers. They are only here to adapt to the new logging method. Also, the new logger that the user implements is the `IdentityLogger` so it will be confusing calling the `LoggerAdapter` `IdentityLogger` while also calling the passes in logger the `IdentityLogger`. 

I think `LoggerAdapterHelper` to `LoggerHelper` makes sense though